python/mxnet/lr_scheduler.py

```diff
@@ -100,16 +100,27 @@ class MultiFactorScheduler(LRScheduler):
 
     Then calculate the new learning rate by::
 
-       base_lr * pow(factor, k+1)
+        base_lr * pow(factor, k+1)
+
+    When warmup_step>1, warmup the learning rate by a const value for first warmup_step steps.
+    It returns a new learning rate by::
+
+        begin_lr + (num_update - 1) * const_update
 
     Parameters
     ----------
     step: list of int
-        The list of steps to schedule a change
+        The list of steps to schedule a change.
     factor: float
         The factor to change the learning rate.
+    warmup_step : int
+        Changes the learning rate for first 'warmup_step' updates.
+    begin_lr : float, optional
+        The learning rate at begin.
+    stop_lr : float, optional
+        Stop updating the learning rate if it is less than this value.
```

Comment:
begin_lr and stop_lr are confusing names and the does doesn't explain exactly what they mean